cmake/modules/FindMaya.cmake

```diff
@@ -210,11 +210,6 @@ set(MAYA_LIBS_TO_FIND
     cgGL
     clew
 )
-if (CMAKE_BUILD_TYPE MATCHES Debug)
-    list(APPEND MAYA_LIBS_TO_FIND tbb_debug)
-else()
-    list(APPEND MAYA_LIBS_TO_FIND tbb)
-endif()
```

Comment:
Even though we were finding TBB from Maya, it was never used from there. Just removed it (the library name changed with onetbb).